src/styles/Home.scss

```diff
@@ -0,0 +1,13 @@
+@import './common/_mixins.scss';
+
+.container {
+    margin-top: rem(120);
+    width: max-content;
```

Comment:
Please don't use this value. It is better to set the width there and autosize children to fit this width.

Reply:
You can call me, I'll describe you more